README.md

```diff
@@ -75,6 +81,80 @@ This assumes you have the correct ssh key in your ssh-agent. If you like to give
 $ docker compose up -d
 ```
 
+### Deploy Serve for local development with Rancher Desktop
+
+Start with instructions in [Serve Charts > How to Deploy](https://github.com/ScilifelabDataCentre/serve-charts?tab=readme-ov-file#how-to-deploy) and come back here when you get to the point of building the studio image.
+
+Again, this setup assumes you have [Rancher Desktop](https://rancherdesktop.io/) installed and running.
+
+```bash
+$ git clone git@github.com:ScilifelabDataCentre/stackn.git
+$ cd stackn
+$ git checkout develop
+$ cp .env.template .env
+$ cp ~/.kube/config cluster.conf
+$ cp ~/.ssh/id_rsa.pub id_rsa.pub
+$ cat Dockerfile local.Dockerfile > local_.dockerfile
+$ nerdctl build --namespace k8s.io -t mystudio -f local_.dockerfile .
+```
+
+Now continue setting up serve charts until you get to the PyCharm setup.
+
+##### PyCharm setup
+
```

Comment:
I think it would be good to mention here that this will not work PyCharm community edition but the licensed version. Maybe we could link to a confluence page where we add information about how to get license for this? Or just say talk to the team.